mteb/tasks/Classification/tha/Thai_Restaurant_Reviews.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification  # type: ignore
+from mteb.abstasks.TaskMetadata import TaskMetadata  # type: ignore
+
+
+class Thai_Restaurant_Reviews(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="Thai_Restaurant_Reviews",
+        description="Wongnai features over 200,000 restaurants, beauty salons, and spas across Thailand on its platform, with detailed information about each merchant and user reviews. In this dataset there are 5 classes corressponding each star rating",
+        reference="https://github.com/wongnai/wongnai-corpus",
+        dataset={
+            "path": "wongnai_reviews",
+            "revision": "e708d4545d7ab10dd2c6b5b5b2a72ca28685dae2",
+        },
+        type="Classification",
+        category="p2p",
+        eval_splits=["train"],
```

Comment:
```suggestion
        eval_splits=["test"],
```
Better use the test split when provided